libs/blocks/merch-card/merch-card.js

```diff
@@ -131,23 +131,41 @@ const getPodType = (styles) => styles?.find((style) => CARD_TYPES.includes(style
 const isHeadingTag = (tagName) => /^H[2-5]$/.test(tagName);
 const isParagraphTag = (tagName) => tagName === 'P';
 
+const appendSlot = (el, slotName, merchCard) => {
+  if (el.length > 0) {
+    const newEl = createTag(
+      'p',
+      { slot: slotName, class: slotName },
+    );
+    el.forEach((e, index) => {
+      newEl.innerHTML += e.innerHTML;
```

Comment:
newEl.appendChild(e.cloneNode(true););

Reply:
This will clone the node with the original tag which is not what I want. I need the HTML to parse it to the new tag to convert from `<h5>` to `<p>`. Since tag names are read-only, the only option is to create a new element and parse the `innerHTML`.